server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/external/datatables/js/jquery.dataTables.js

```diff
@@ -10514,7 +10514,7 @@
 		 *      } );
 		 *    } );
 		 */
-		"aLengthMenu": [ 10, 25, 50, 100 ],
+		"aLengthMenu": [[10, 25, 50, 100, -1], [10, 25, 50, 100, "All"]],
```

Comment:
I don't think this is the best place to make this change. We would need to remember to manually redo this everytime we upgrade this external resource. 

I tried out another way to do this which worked well and will persist across updates to jquery we might make.

Added this at the top of `functions.js` and it worked as expected:
```javascript
if ($.fn && $.fn.dataTable) {
  $.extend(true, $.fn.dataTable.defaults, {
    "lengthMenu": [
      [10, 25, 50, 100, -1],
      [10, 25, 50, 100, "All"]
    ]
  });
}
```